shared/javascripts/network_service.js

```diff
@@ -446,6 +446,7 @@ var privlyNetworkService = {
     $(".logged_out_nav").show();
     $(".injected_hide").show();
     privlyNetworkService.mobileHide();
+    androidJsBridge.showLoginActivity();
```

Comment:
You should check if the application is running on the android platform. It does not make sense to do this on another platform.